Adding graph from model_config (#3094)
openvinotoolkit/model_server: docs/mediapipe.md

### Starting OpenVINO Model Server with Mediapipe servables
MediaPipe servables configuration is to be placed in the same json file like the
[models config file](starting_server.md).
Both models and graphs can be defined in section `model_config_list`.

> In this case `name` and `base_path` parameters are required and `graph_path` parameter is optional for custom `graph.pbtxt` names.

Reviewer: ```suggestion
In this case `name` and `base_path` parameters are required and `graph_path` parameter is optional for custom graph configuration file names (other than default `graph.pbtxt`).
```

Reviewer: @michalkulakowski

> Just a `config.json` needs to be prepared with a list of all the graphs to be deployed:

Reviewer: Add spacing and rewrite suggestion:
```suggestion

Here is example `config.json` file that defines two MediaPipe graphs.
```

```json
{
    "model_config_list": [
        {
            "config": {
                "name": "mediapipe_graph_name",
                "base_path":"/custom/path/to/pbtxt",
                "graph_path": "custom_name_for.pbtxt"
            }
        },
        {
            "config": {
                "name": "mediapipe_graph_name_for_default_name",
                "base_path":"/custom/path/to/pbtxt"
            }
        }
    ]
}
```

> In case the `mediapipe_graph_name_for_default_name` above, ovms will search for the default graph name `graph.pbtxt` in the `/custom/path/to/pbtxt` directory.

Reviewer: I dont think this sentence is needed if you take my suggestions above - already explained above

This will load the `mediapipe_graph_name_for_default_name` servable from `/custom/path/to/pbtxt/graph.pbtxt` path.

In case the `mediapipe_graph_name` above, ovms will search for the graph name `custom_name_for.pbtxt` in the `/custom/path/to/pbtxt` directory.
This will load the `mediapipe_graph_name` servable from `/custom/path/to/pbtxt/custom_name_for.pbtxt` path.

> Graphs can also be configured inthe `mediapipe_config_list` section.

Reviewer 2: ```suggestion
Graphs can also be configured in the `mediapipe_config_list` section.
```

Reviewer: + add info that it is deprecated?

Reviewer: Graphs can also be configured inthe `mediapipe_config_list` section, however - this is deprecated.

> When the MediaPipe graphs artifacts are packaged like presented above, configuring the OpenVINO Model Server is very easy.
> Just a `config.json` needs to be prepared with a list of all the graphs to be deployed:

Reviewer: ```suggestion
When using legacy field, `base_path` can be omitted. It will be deduced from ...
```

from where BTW?
